src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,21 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int weight;
+
     public Excavator() {
     }
 
+    public Excavator(String name, String colour, int weight) {
+        this.setName(name);
+        this.setColor(colour);
```

Comment:
another option is - creating a constructor in a Machine class with name and color and using the `super(name, color)` here